docs/3.0rc/resources/upgrade-prefect-3.mdx

```diff
@@ -0,0 +1,224 @@
+    @flow
+    def my_flow():
+	    return my_task()
+	
+		
+    if __name__ == "__main__":
+	    my_flow()
+    ```
+  </Card>
+  <Card title="Prefect 3">
+    ```python
+    @task
+    async def my_task():
+	    ...
+	
+
+    # Enclosing flow must be async as well
+    @flow
+    async def my_flow():
+	    return await my_task()
+	
+	
+    if __name__ == "__main__":
+	    asyncio.run(my_flow())
+    ``` 
+  </Card>
+</CardGroup>
+
+Because `Task.submit` is always a synchronous call, it does not need to be awaited. 
+`Task.submit` is still non-blocking and execution of the submitted task is also still non-blocking. 
+So, *you do not need to await futures from submitted tasks*:
+
+<CardGroup cols={2}>
+  <Card title="Prefect 2">
+    ```python
+    @task
+    async def my_task():
+	    ...
+	
+	
+    @flow
+    async def my_flow():
+	    #must await `.submit()`
+	    future = await my_task.submit()
+	    return future
+    ```
+  </Card>
+  <Card title="Prefect 3">
+    ```python
+    @task
+    async def my_task():
+	    ...
+	
+	
+    @flow
+    async def my_flow():
+	    # No need to await `.submit()`
+	    future = my_task.submit()
+	    return future
+    ```
+  </Card>
+</CardGroup>
+
+The `PrefectFuture` methods are also always synchronous in Prefect 3. Names and return values have changed to reflect this:
+
+- `PrefectFututure.get_state()` is now `PrefectFuture.state`
+- `PrefectFuture.wait()` now returns `None` instead of a `State`
+
+*Wait for the future to complete, then retrieve the state separately*:
+
+<CardGroup cols={2}>
+  <Card title="Prefect 2">
+    ```python
+    @task
+    async def my_task():
+    	    ...
+	
+	
+    @flow
+    async def my_flow():
+    	future = await my_task.submit()
+	    # State is returned by `wait()`
+	    state = await future.wait()
+	    return state
+    ```
+  </Card>
+  <Card title="Prefect 3">
+    ```python
+    @task
+    async def my_task():
+    	    ...
+	
+	
+    @flow
+    async def my_flow():
+	    future = my_task.submit()
+	    # Wait for future completion  
+	    # before retrieving the state
+	    future.wait()
+	    state = future.state
+	    return state
+    ```
+  </Card>
+</CardGroup>
+
+In Prefect 3, `PrefectFutures` resulting from submitted task runs are not automatically awaited/resolved before exiting a flow, unless they’re a dependency of other flows or tasks, so *you must wait for a task’s `PrefectFuture` resolution before the end of a flow*:
+
+<CardGroup cols={2}>
+  <Card title="Prefect 2">
+    ```python
+    @task
+    def my_task():
+	    ...
+	
+	
+    @flow
+    def my_flow():
+	    # The flow will not exit until these 
+	    # futures are resolved
+	    [my_task.submit() for i in range(10)]
+    ```
+  </Card>
+  <Card title="Prefect 3">
+    ```python
+    @task
+    def my_task():
+	    ...
+	
+	
+    @flow
+    def my_flow():
+	    futures = [my_task.submit() for i in range(10)]
+	    # You must wait for futures before exiting flow.
+	    futures.wait()
```

Comment:
This won't work as `futures` is a standard python list here; if you use `.map` or the soon-to-be `wait(*futures)` utility this will still look clean